.github/workflows/deploy_artifacts.yml

```diff
@@ -75,7 +75,7 @@ jobs:
       - name: Generate database documentation  
         run: | 
           rm dockstore-webservice/target/dockstore-webservice-*sources.jar || true
-          java -jar dockstore-webservice/target/dockstore-webservice-*.jar db migrate --include 1.3.0.generated,1.3.1.consistency,1.4.0,1.5.0,1.6.0,1.7.0,1.8.0,1.9.0,1.10.0,1.11.0,1.12.0,1.13.0,1.14.0,1.15.0,1.16.0  dockstore-integration-testing/src/test/resources/dockstore.yml
+          java -jar dockstore-webservice/target/dockstore-webservice-*.jar db migrate --include 1.3.0.generated,1.3.1.consistency,1.4.0,1.5.0,1.6.0,1.7.0,1.8.0,1.9.0,1.10.0,1.11.0,1.12.0,1.13.0,1.14.0,1.15.0,1.16.0,1.17.0  dockstore-integration-testing/src/test/resources/dockstore.yml
```

Comment:
Will need to add a compose_setup PR to include this migration https://github.com/dockstore/compose_setup/blob/a14cd1d390f4ae1e14a8ba6e36ec06ce5fe2604e/templates/init_migration.sh.template#L15 and update the compose_setup submodule in dockstore-deploy